lang/python/python-pynacl/Makefile

```diff
@@ -1,17 +1,17 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python-pynacl
-PKG_VERSION:=1.5.0
-PKG_RELEASE:=3
+PKG_VERSION:=1.6.1
+PKG_RELEASE:=1
 
-PYPI_NAME:=PyNaCl
-PKG_HASH:=8ac7448f09ab85811607bdd21ec2464495ac8b7c66d146bf545b0f08fb9220ba
+PYPI_NAME:=pynacl
+PKG_HASH:=8d361dac0309f2b6ad33b349a56cd163c98430d409fa503b10b70b3ad66eaa1d
 
 PKG_MAINTAINER:=Javier Marcet <javier@marcet.info>
 PKG_LICENSE:=Apache-2.0
 PKG_LICENSE_FILES:=LICENSE
 
-PKG_BUILD_DEPENDS:=libffi/host python-cffi/host  # cffi>=1.4.1
+PKG_BUILD_DEPENDS:=python3-setuptools/host libffi/host python-cffi/host  # cffi>=1.4.1
```

Comment:
`python3-setuptools/host` -> `python-setuptools/host`

I guess it's a typo error?